handsontable/src/core.js

```diff
@@ -3350,11 +3350,13 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
    * @function getCellMeta
    * @param {number} row Visual row index.
    * @param {number} column Visual column index.
+   * @param {object} options Execution options for the `getCellMeta` method.
+   * @param {boolean} [options.skipMetaExtension] If `true`, skips extending the cell meta object. Default: `false`.
```

Comment:
What do you think about more specifying the description? Something like
```suggestion
   * @param {boolean} [options.skipMetaExtension=false] If `true`, skips extending the cell meta object, so the `cells` function won't be called as the `afterGetCellMeta` and `beforeGetCellMeta` hooks.
```

Reply:
Done